amt/api/routes/project.py

```diff
@@ -244,12 +268,23 @@ async def get_system_card_requirements(
         request,
     )
 
+    # TODO: This is only for the demo of 18 Oct. In reality one would load the requirements from the requirement
```

Comment:
is this TODO still relevant?

Reply:
Yeah true, we already replaced with your implementation of the system_card storage, so we can remove it. But then just for the comment I would not want to run the whole pipeline and waiting for review again if this is the only remark